js/module.js

```diff
@@ -415,3 +415,7 @@ Module.register = function (name, moduleDefinition) {
 	Log.log("Module registered: " + name);
 	Module.definitions[name] = moduleDefinition;
 };
+
+exports._test = {
```

Comment:
PR #633 removed the error for me and `npm test` works. Seems like a good solution to me.

Also noticed a minor textual improvement in `tests/functions/compare-version.js`:
"Test function ... **into** ..." could better be "Test function ... **in** ...".